python/tvm/auto_scheduler/utils.py

```diff
@@ -306,6 +307,7 @@ def _func_wrapper(que, func, args, kwargs, add_thread_wrapper):
 def call_func_with_timeout(timeout, func, args=(), kwargs=None, add_thread_wrapper=False):
     """Call a function with timeout"""
     que = multiprocessing.Queue(2)
+    process = PopenWorker()
```

Comment:
Should this multiprocessing also be changed to PopenPoolWorker? This change is similar to #7889

Reply:
yes, and `PopenWorker` supports for `timeout` argument in `send` which is a builtin mechanism of calling with timeout, so queue is no longer needed, this code block can be simplified

Reply:
argument `add_thread_wrapper` can be removed, it uses `popen` to create a new process which won't cause conflict with tvm thread pool. cc @merrymercy @jcf94

Reply:
Done.